packages/site-kit/components/docs/Main.svelte

```diff
@@ -171,45 +75,8 @@
 		-moz-tab-size: 2;
 	}
 
-	@media (min-width: 832px) { /* can't use vars in @media :( */
-		aside {
-			display: block;
-			width: var(--sidebar-w);
-			height: 100vh;
-			top: 0;
-			left: 0;
-			overflow: hidden;
-			box-shadow: none;
-			border: none;
-			overflow: hidden;
-			background-color: var(--second);
-			color: white;
-		}
-
-		aside.open::before {
-			display: none;
-		}
-
-		aside::after {
-			content: '';
-			bottom: 0;
-			height: var(--top-offset);
-			background: linear-gradient(to bottom, rgba(103,103,120,0) 0%, rgba(103,103,120,0.7) 50%, rgba(103,103,120,1) 100%);
-		}
-
-		aside button {
-			display: none;
-		}
-
-		.sidebar {
-			padding: var(--top-offset) 0 6.4rem 3.2rem;
-			font-family: var(--font);
-			overflow-y: auto;
-			height: 100%;
-			bottom: auto;
-			width: 100%;
-		}
-
+	@media (min-width: 832px) {
+		/* can't use vars in @media :( */
```

Comment:
I think you can with `env`? But not my area of expertise

https://developer.mozilla.org/en-US/docs/Web/CSS/env()

Reply:
🤯

Reply:
Could use a [postcss plugin](https://github.com/postcss/postcss-custom-media) for this, it follows the spec.

Reply:
Personally I'm inclined towards the magic numbers just because it's one less moving part. That comment has been there for a long time, and we've managed